operator/src/main/java/io/github/microcks/operator/base/resources/MicrocksDeploymentDependentResource.java

```diff
@@ -137,6 +137,20 @@ protected Deployment desired(Microcks microcks, Context<Microcks> context) {
                .endTemplate()
             .endSpec();
 
+      microcks.getSpec().getMicrocks().getEnv().forEach(env -> {
+         builder.editSpec()
+               .editTemplate()
+                 .editSpec()
+                    .editFirstContainer()
+                        .addNewEnv()
+                            .withName(env.getName())
+                            .withValue(env.getValue())
+                        .endEnv()
+                    .endContainer()
+                 .endSpec()
+               .endTemplate().endSpec();
+      });
+
```

Comment:
There's a `.addAllToEnv(spec.getMicrocks().getEnv())` method that can be used and that is less verbose. Would you mind applying this within the first container editing session? Somewhere around line 101 would be perfect.